packages/console/src/pages/Applications/components/GetStarted/index.tsx

```diff
@@ -20,10 +26,49 @@ type Props = {
   onClose: () => void;
 };
 
+type DocumentFileNames = {
+  files: string[];
+};
+
+type Step = {
+  title?: string;
+  subtitle?: string;
+  metadata: string;
+};
+
 const GetStarted = ({ appName, onClose }: Props) => {
   const [isLibrarySelectorFolded, setIsLibrarySelectorFolded] = useState(false);
-  const [libraryName, setLibraryName] = useState<string>(SupportedJavascriptLibraries.Angular);
+  const [libraryName, setLibraryName] = useState<string>(SupportedJavascriptLibraries.React);
+  const [activeStepIndex, setActiveStepIndex] = useState<number>(-1);
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+  const publicPath = useMemo(
+    () => `/console/get-started/${libraryName}/${i18next.language}`,
+    [libraryName]
+  );
+
+  const { data: jsonData } = useSWRImmutable<DocumentFileNames>(`${publicPath}/index.json`);
+  const { data: steps } = useSWRImmutable<Step[]>(jsonData, async ({ files }: DocumentFileNames) =>
```

Comment:
Just curious, can we just use `useSWR`?

Reply:
useSWR will end up re-fetching the markdown files every time browser gets focused